spec/system/documents/edit_numbers_spec.rb

```diff
@@ -12,30 +12,12 @@
   let(:assessor) { create :user, :assessor, local_authority: local_authority }
   let(:reviewer) { create :user, :reviewer, local_authority: local_authority }
 
-  context "as a user who is not logged in" do
```

Comment:
Maybe this file just needs renaming, or maybe it needs deleting altogether? But seeing as "Edit document numbers page" doesn't exist any more, what is this testing, and is it tested elsewhere?

Reply:
I think renaming is the way to go. This tests an area of functionality that is not tested elsewhere (verifying that numbers can be added to documents at varying points in the journey and that those numbers then appear on the decision notice) so we need to keep it BUT the current filename sounds like we are testing the now-deleted route. I will change it to `numbering_documents.spec` for clarity.